cpp/src/aztec3/circuits/rollup/base/.test.cpp

```diff
@@ -593,30 +610,37 @@ TEST_F(base_rollup_tests, test_compute_membership_historic_private_data)
         .sibling_path = sibling_path,
     };
 
-    BaseOrMergeRollupPublicInputs outputs = aztec3::circuits::rollup::native_base_rollup::base_rollup_circuit(inputs);
+    BaseOrMergeRollupPublicInputs outputs =
+        aztec3::circuits::rollup::native_base_rollup::base_rollup_circuit(composer, inputs);
 }
 
 TEST_F(base_rollup_tests, test_constants_dont_change)
 {
+    DummyComposer composer = DummyComposer();
     BaseRollupInputs inputs = dummy_base_rollup_inputs_with_vk_proof();
-    BaseOrMergeRollupPublicInputs outputs = aztec3::circuits::rollup::native_base_rollup::base_rollup_circuit(inputs);
+    BaseOrMergeRollupPublicInputs outputs =
+        aztec3::circuits::rollup::native_base_rollup::base_rollup_circuit(composer, inputs);
     ASSERT_EQ(inputs.constants, outputs.constants);
     run_cbind(inputs, outputs);
 }
 
 TEST_F(base_rollup_tests, test_aggregate)
 {
     // TODO: Fix this when aggregation works
+    DummyComposer composer = DummyComposer();
     BaseRollupInputs inputs = dummy_base_rollup_inputs_with_vk_proof();
-    BaseOrMergeRollupPublicInputs outputs = aztec3::circuits::rollup::native_base_rollup::base_rollup_circuit(inputs);
+    BaseOrMergeRollupPublicInputs outputs =
+        aztec3::circuits::rollup::native_base_rollup::base_rollup_circuit(composer, inputs);
     ASSERT_EQ(inputs.kernel_data[0].public_inputs.end.aggregation_object.public_inputs,
               outputs.end_aggregation_object.public_inputs);
 }
 
 TEST_F(base_rollup_tests, test_subtree_height_is_0)
 {
+    DummyComposer composer = DummyComposer();
     BaseRollupInputs inputs = dummy_base_rollup_inputs_with_vk_proof();
-    BaseOrMergeRollupPublicInputs outputs = aztec3::circuits::rollup::native_base_rollup::base_rollup_circuit(inputs);
+    BaseOrMergeRollupPublicInputs outputs =
```

Comment:
Looks like there aren't any negative tests for the base rollup yet. Something for next week :)